doc/stake-pool-operations/retire_stakepool.md

```diff
@@ -2,8 +2,17 @@
 
 To retire a pool we need to:
 
-* Create a **deregistration certificate** and
-* Submit the certificate to the blockchain with a **transaction**
+1. Create a **deregistration certificate**
```

Comment:
I would call this a retirement certificate.

Reply:
Makes sense to me, however this is how it is called on the CLI, using a different name might lead to confusion.  

```
cardano-cli stake-pool
Usage: cardano-cli stake-pool (registration-certificate | 
                                deregistration-certificate | id | metadata-hash)
  Stake pool commands

Available options:
  -h,--help                Show this help text

Available commands:
  registration-certificate Create a stake pool registration certificate
  deregistration-certificate
                           Create a stake pool deregistration certificate
  id                       Build pool id from the offline key
  metadata-hash            Print the hash of pool metadata.
```

Reply:
ah, consistency with the CLI does seem better!